spec/models/experiment_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe Experiment, type: :model do
+  it { should have_many :scientist_experiments }
+  it { should have_many(:scientists).through(:scientist_experiments) }
+
+
+  # Model test for long_running_experiments
```

Comment:
If we're following a TDD approach, the test for `longest_running_experiments` should be created before the implementation code.